### PR TITLE
Bug 1836801: Filter Disk Import activity that does not have progress yet

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/plugin.tsx [hunk 1]
```diff
@@ -377,7 +377,8 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
       isActivity: (resource) =>
         resource?.status?.phase === 'ImportInProgress' &&
-        !!resource?.metadata?.ownerReferences?.[0],
+        !!resource?.metadata?.ownerReferences?.[0] &&
+        !Number.isNaN(parseInt(resource?.status?.progress, 10)),
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/plugin.tsx:381]: btw, N/A progress might be a valid option for sources where CDI doesn't have progress implemented.

The best solution would be to check importer pod status (if it crashloops) - which is quite a heavy operation though.

Can't we just believe the status of a CRD (treat it as API) and show 0 or `N/A` progress instead?

Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/plugin.tsx]: Checking pods is not really an option - as you mentioned - too heavy.
I've opted for showing a generic activity without progress bar if we cannot parse the progress info
Screenshot from 2020-05-18 14-31-23

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/plugin.tsx]: +1

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-  </ActivityProgress>
-);
+  );
+  return Number.isNaN(progress) ? (
+    <>
+      <ActivityItem>{VM_IMPORT_TITLE}</ActivityItem>
+      {vmLink}
+    </>
+  ) : (
+    <ActivityProgress title={VM_IMPORT_TITLE} progress={progress}>
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx:28]: can we include a key?

Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx:28]: why ? Adding key is handled by ActivityBody https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx#L160

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx:28]: ah, sorry - no need to have it here

**Pull request merged**